docs/index.md

```diff
@@ -0,0 +1,58 @@
+# < Charm name > Operator
+
+<!-- A single sentence that says what the product is, succinctly and memorably. Add a 1-2 sentence description of what the charm software does. -->
+
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing < Charm software > on 
+Kubernetes. 
+
+<!-- A paragraph of 2-5 short sentences, that describes what the product does and what need the product meets. -->
+
+Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling, and more. 
+For < Charm name >, this includes:
+* list or summary of app-specific features
+
+The < Charm name > charm allows for deployment on many different Kubernetes platforms, from [MicroK8s](https://microk8s.io/) to 
+[Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes offerings.
+
+<!-- Finally, a paragraph that describes whom the product is useful for. -->
+
+This charm will make operating < Charm software > simple and straightforward for DevOps or SRE teams through Juju's clean interface. 
+
+## In this documentation
+
+| | |
+|--|--|
+|  [Tutorials](link to tutorial)</br>  Get started - a hands-on introduction to using the charm for new users </br> |  [How-to guides](link to how-to guide) </br> Step-by-step guides covering key operations and common tasks |
+| [Reference](link to reference) </br> Technical information - specifications, APIs, architecture | [Explanation](link to explanation) </br> Concepts - discussion and clarification of key topics  |
+
+## Contributing to this documentation
+
+Documentation is an important part of this project, and we take the same open-source approach to the documentation as 
+the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. 
+Our documentation is hosted on the [Charmhub forum](https://discourse.charmhub.io/) 
+to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to 
+either directly change something you see that's wrong, ask a question or make a suggestion about a potential change via 
+the comments section.
+
+If there's a particular area of documentation that you'd like to see that's missing, please 
+[file a bug](link to issues page).
+
+## Project and community
+
+The < Charm > Operator is a member of the Ubuntu family. It's an open-source project that warmly welcomes community 
```

Comment:
Maybe replace `< Charm >` with `< Charm name >` here so that it is consistent with the previous occurrences. It would make search and replace easier

Reply:
Good catch, thanks!